content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Operations Start Guide
+type: docs
+---
+
+## Introduction
+
+The Operations Start Guide will help you start with EGI Operations duties. It
+covers the responsibilities of the various parties involved in the running of
+the [EGI infrastructure](https://ims.egi.eu/display/EGIG/EGI+Infrastructure) and
+guide how to join operations. As a newcomer, you need to understand the
+structure of the infrastructure and roles of operators at different levels.
+Reading the whole document will give you a complete overall picture of daily
+operations within EGI.
+
+## Resource Centres and Resource Infrastructures
+
+Resources are geographically distributed and are contributed by
+[Resource Centres](https://ims.egi.eu/display/EGIG/Resource+Centre). A Resource
+Centre is the smallest resource administration domain within EGI. It can be
+either localized or geographically distributed. A Resource Centre is also known
+as a _site_. It provides a minimum set of local or remote IT Services, such as
+High Throughput Compute (HTC), Cloud Compute, Storage and Data Transfer,
+compliant to well-defined IT Capabilities necessary to make resources accessible
+to authorised users. Users can access the services using common interfaces.
+
+A
+[Resource Infrastructure](https://ims.egi.eu/display/EGIG/Resource+Infrastructure)
```

Comment:
@paolini78, @mviljoen-egi is this a generic/convoluted way to say that the EGI Infrastructure federates the resources from the Resource Infrastructure Providers? Maybe we should make it less generic.

Reply:
I have only tried to give a quick overview of the structure.

Reply:
OK, I will push an update to try to simplify it.

Reply:
Done via https://github.com/EGI-Federation/documentation/pull/323/commits/9f7c2889ba21d0c3756eb869a74034121a57d125